src/libraries/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -518,14 +518,9 @@ public abstract partial class JsonConverter
     public partial class JsonConverterAttribute : System.Text.Json.Serialization.JsonAttribute
     {
         protected JsonConverterAttribute() { }
-#if NETCOREAPP && !NETCOREAPP3_0
```

Comment:
**Question:** How come this was caught, but not the #if/def above 

```C#
public static partial class JsonSerializer {
#if NETCOREAPP && !NETCOREAPP3_0
...
```

Reply:
I think it was just my mistake when looking at the diff and I accidentally discarded that change and then gave it a thought but forgot to redo the removal.